src/controllers/selectedAccount/selectedAccount.ts

```diff
@@ -447,6 +522,7 @@ export class SelectedAccountController extends EventEmitter {
       ...this,
       ...super.toJSON(),
       firstCashbackBanner: this.firstCashbackBanner,
+      cashbackStatusByAccount: this.cashbackStatus,
```

Comment:
Why is that? `cashbackStatusByAccount` can be used directly. This pattern is only used for getters.